Become a Carpentries Partner (#666)
carpentries/new-website: content/posts/2026/04/become-a-carpentries-partner.md

Through hands-on workshops, Instructor training, and collaborative lesson development, our global community supports organisations in building practical skills for working with data and code while fostering a culture of open, reproducible research.

Today, the Carpentries community includes:

- **5,413 Instructors**
- **185 Instructor Trainers**
- and **4,968 workshops taught**
- across **72 countries**

But the real impact happens when organisations build their own local communities of practice around these skills.

## Who Carpentries Partnerships Are For

Carpentries Partners come from a wide range of organisations, including:

- universities and research institutes  
- government laboratories  
- library consortia  
- research software engineering groups  
- non-profit organisations and scientific networks.  

Many Partners begin their engagement with a single workshop or training cohort and expand over time as their local community grows.

## What a Carpentries Partnership Provides

> The Carpentries Partnership Program is designed to help organisations grow sustainable training capacity while connecting to a global community of educators and researchers.

Reviewer: ```suggestion
The Carpentries Partnership program is designed to help organisations grow sustainable training capacity while connecting to a global community of educators and researchers.
```

Partnerships can include combinations of:

- **Centrally-Organised workshops** using Carpentries data and computational skills curricula  

> - **Instructor Training** to prepare new instructors in evidence-based teaching practices  

Reviewer: ```suggestion
- **Instructor Training** to prepare new Instructors in evidence-based teaching practices  
```

- **Collaborative Lesson Development Training** for teams building open educational resources  
- **Professional development short courses** to build timely skills for the modern digital workforce  
- **Consultation and support** for building local communities of practice.  

Rather than relying on external training, our Partners use these programs to develop internal capacity, empowering Instructors, curriculum developers, and community builders and supporting ongoing skill development within their organisations.

## A Global Community of Educators and Researchers

> Beyond the training programmes themselves, partnering with The Carpentries [connects organisations to a global network](https://carpentries.org/blog/2026/04/why-partner-with-the-carpentries/) of Instructors, Maintainers, Instructor Trainers, and community leaders.

Reviewer: ```suggestion
Beyond the training programs themselves, partnering with The Carpentries [connects organisations to a global network](https://carpentries.org/blog/2026/04/why-partner-with-the-carpentries/) of Instructors, Maintainers, Instructor Trainers, and community leaders.
```

Author: This is one of our peculiarities; our Lesson Programs (and Partnership program) are referred to as "program/s" but the rest of our activities conform to our general style guide (i.e., programme/s), so won't be changing this @ErinBecker.

This network shares teaching practices, develops open lesson materials, and collaborates across institutions and disciplines to support better research and learning.

As one community member shared:

> “I went from a learner to Instructor and completely changed the trajectory of my career. It helped me find my people.”  
— Dr. Saranya Canchi, Data Engineer - Bioinformatics, Howard Hughes Medical Institute

[Such stories from The Carpentries community](https://carpentries.org/blog/2026/04/who-are-the-carpentries/) illustrate how technical skills training can also foster belonging, confidence, and collaboration.

## Take the Next Step

Whether your organisation is exploring a partnership for the first time or looking to expand an existing collaboration, we would be happy to talk with you about your goals.

You can:

**Become a Partner**  
Join the [Carpentries Partnership program](https://carpentries.org/support/partnership/) and begin building data and computational skills within your organisation.

**Renew Your Partnership**  
Continue your organisation’s engagement and expand the impact of your local Carpentries community.

**Schedule a Consultation**  

> [Meet with a member of the Carpentries team](https://calendly.com/carpentries-partnership/onboarding) to discuss how a partnership can support your organisation’s training and capacity-building goals.

Reviewer: ```suggestion
[Meet with a member of The Carpentries team](https://calendly.com/carpentries-partnership/30-minutes) to discuss how a partnership can support your organisation’s training and capacity-building goals.
```

Reviewer: The "onboarding" link is for organisations who have already signed Partnerships. Not a huge deal, but keeping the uses separate helps team members prepare for meetings based on what meeting type shows up on their calendar.

## Start the Conversation

> The Carpentries Partnership Program is designed to be flexible so that organisations can build the combination of services that best supports their community.

Reviewer: ```suggestion
The Carpentries Partnership program is designed to be flexible so that organisations can build the combination of services that best supports their community.
```

> If you are interested in becoming a partner, renewing your partnership, or learning more about how the programme works, we invite you to connect with us.

Reviewer: ```suggestion
If you are interested in becoming a Partner, renewing your partnership, or learning more about how the program works, we invite you to connect with us.
```

> Learn more and [schedule a consultation](mailto:membership@carpentries.org).

Reviewer: ```suggestion
Learn more and [schedule a consultation](mailto:partnership@carpentries.org).
```
